controllers/progressivesync_controller.go

```diff
@@ -211,14 +212,15 @@ func (r *ProgressiveSyncReconciler) requestsForSecretChange(o client.Object) []r
 		for _, app := range appList.Items {
 			if app.Spec.Destination.Server == string(s.Data["server"]) && pr.Owns(app.GetOwnerReferences()) {
 				/*
-					Consider the following scenario:
-					- 2 Applications
-					- owned by the same ApplicationSet
-					- referenced by the same Progressive Rollout
-					- targeting the same cluster
-
-					In this scenario, we would trigger the reconciliation loop twice.
-					To avoid that, we use a map to store for which Progressive Rollout objects we already trigger the reconciliation loop.
+						Consider the following scenario:
```

Comment:
did `go fmt` triggered that tabbing here?

Reply:
Yes, I'm not sure why to be honest.

Reply:
I can revert it if we want.

Reply:
i was just curious :-)